browsers/private/update-tool/download_helpers.mts

```diff
@@ -35,19 +34,19 @@ export async function downloadFileThroughStreaming(
     const stream = Readable.fromWeb(result.body);
     const outStream = fs.createWriteStream(destinationPath);
 
-    stream.on('error', err => reject(err));
-    stream.on('close', () => resolve());
+    stream.on("error", (err) => reject(err));
+    stream.on("close", () => resolve());
     stream.pipe(outStream);
   });
 }
 
 export async function sha256(filePath: string): Promise<string> {
   return new Promise((resolve, reject) => {
     const stream = fs.createReadStream(filePath);
-    const hash = createHash('sha256');
+    const hash = createHash("sha256");
 
-    stream.on('error', err => reject(err));
-    stream.on('close', () => resolve(hash.digest('hex')));
+    stream.on("error", (err) => reject(err));
```

Comment:
Nit: we should adjust prettier config to use single quotes

Reply:
Done in #25.